docs/api/app.md

```diff
@@ -890,6 +890,15 @@ technologies, such as screen readers, has been detected. See
 https://www.chromium.org/developers/design-documents/accessibility for more
 details.
 
+### `app.setAccessibilitySupportEnabled(value)` _macOS_ _Windows_
```

Comment:
I would suggest using a more descriptive name than `value` for the boolean. Something like `enabled` would be more consistent with other APIs that accept boolean arguments:

- `win.setResizable(resizable)`
- `win.setMovable(movable)`
- `win.setMinimizable(minimizable)`

Reply:
Oh, it was `enabled` initially but then I decided to follow the `app.commandLine.appendArgument(value)` nearby. Fixed!